submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/wiremock/FireMockDatabase.kt

```diff
@@ -23,14 +24,18 @@ class FireMockDatabase(
         return fireFile
     }
 
-    fun setPath(fireOid: String, firePath: String) {
-        if (recordsById.values.any { it.firePath == firePath })
+    fun setPath(fireOid: String, firePath: String): FireApiFile {
+        if (recordsById.values.any { it.firePath == firePath }) {
             throw FireException("Path '$firePath' is already allocated", CONFLICT)
+        }
 
-        val record = recordsById.getValue(fireOid)
-        recordsById[fireOid] = record.copy(firePath = firePath)
+        val record = recordsById.getValue(fireOid).copy(firePath = firePath)
```

Comment:
why this changes?  I mean I understand how this implementation will never work for a real world but in our test when we try to run code that generate issues here?

Reply:
The real FIRE api returns a `FireApiFile` when setting the path so I'm expecting that in the `FireWebClient`.  I just made this change in order to keep the mock consistent with the real api